assets/src/components/resizable-box/edit.css

```diff
@@ -21,6 +21,11 @@
 .wp-block-image .components-resizable-box__container img {
 	height: 100%;
 	object-fit: cover;
+	/* Prevents dragging the image itself, as its container is draggable. */
+	-webkit-user-drag: none;
```

Comment:
Pretty sure that only `-webkit-user-drag` exists nowadays. The other properties either never existed (because it was never in the spec) or are simply irrelevant.

Reply:
Thanks, good point.

Reply:
https://github.com/ampproject/amp-wp/pull/2389/commits/cae00b77a931eb23fe87ad298d1c73e2113085e4 removes these properties. You're right, I can't find any reference to them with quick searches.